### PR TITLE
Add coverage for wrapper[Symbol.iterator]

### PR DESCRIPTION
**Pull request by megawac**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -42,6 +42,7 @@
   var phantom = root.phantom,
       amd = root.define && define.amd,
       argv = root.process && process.argv,
+      jsArrayFrom = Array.from,
```
Comment (jdalton) [inline test/test.js]: No need for this. You can use `_.toArray(wrapped)`.

Comment (megawac) [inline test/test.js]: I thought it'd be better to ensure it conforms to an actual browser implementation

File: test/test.js [hunk 2]
```diff
@@ -16598,7 +16599,7 @@
 
     var chainType = ' in an ' + (implict ? 'implict' : 'explict') + ' chain';
 
-    test('should produce an ES6 compliant object' + chainType, 7, function() {
+    test('should follow ES2015 iterator specification' + chainType, 7, function() {
```
Comment (jdalton) [inline test/test.js]: Don't worry about the test label here. I'm changing it in a cleanup pass.

File: test/test.js (hunk 1 above)
Comment (jdalton) [inline test/test.js]: `_.toArray` conforms to it so that's fine and lets it be tested in Chrome and Node which lacks `Array.from`.

Comment (jdalton) [inline test/test.js]: I still need to add unit tests for `_.toArray` and the like (more for this evening).

**Comment by jdalton** [comment]
Thanks Graeme!

**Pull request merged**